plugins/TodayAtMun/__init__.py

```diff
@@ -0,0 +1,58 @@
+import discord
+from discord.ext import commands
+
+from Bot import bot
+from Plugin import AutomataPlugin
+from plugins.TodayAtMun.DiaryParser import DiaryParser
+from plugins.TodayAtMun.Today import Today
```

Comment:
```suggestion
from plugins.TodayAtMun.today import Today
```

file is named `today`, not `Today` so import fails on my machine, maybe to keep it consistent with the rest of the project, rename `DiaryParser.py` to `diary_parser` instead?

Reply:
also a note: I assume you're writing this code on MacOS, since for some lovely reason apple has chosen to have a cAse insEnsitIve filesystem, by DEFAULT, so this would work fine for you, but on no other sane setup :D

Reply:
I’m on windows actually 😳 I should probably just dual boot my desktop pc